content/collections/api/en/chart-annotations.md

```diff
@@ -4,122 +4,588 @@ blueprint: api
 title: 'Chart Annotations API'
 source: 'https://www.docs.developers.amplitude.com/analytics/apis/chart-annotations-api/'
 auth_method: http_basic
-standard_endpoint: 'https://amplitude.com/api/2/annotations'
-eu_endpoint: 'https://analytics.eu.amplitude.com/api/2/annotations'
+standard_endpoint: 'https://amplitude.com/api/3/annotations'
+eu_endpoint: 'https://analytics.eu.amplitude.com/api/3/annotations'
 postman_link: 'https://www.postman.com/amplitude-dev-docs/workspace/amplitude-developers/folder/20044411-c858ea6d-9201-4acd-b0aa-d78692da1b44?action=share&source=copy-link&creator=29131806&ctx=documentation'
-lede: "The Chart Annotations API lets you programmatically annotate important dates like feature releases and marketing campaigns on your organization's charts with a horizontal axis of calendar dates."
+lede: "The Chart Annotations API lets you programmatically annotate important dates and time ranges like feature releases and marketing campaigns on your organization's charts. You can organize annotations into categories and use hourly granularity for precise timing."
 updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
 updated_at: 1716312268
 api_status: ga
-summary: "Programmatically annotate important dates like feature releases and marketing campaigns on your organization's charts with a horizontal axis of calendar dates."
+summary: "Programmatically annotate important dates and time ranges like feature releases and marketing campaigns on your organization's charts. Organize annotations into categories and use hourly granularity for precise timing."
 ---
-## Create an annotation
+## Annotation categories
 
+Manage annotation categories to organize your annotations. Create categories, update their names, and assign them to annotations.
+
+### Create an annotation category
+
+{{partial:tabs tabs="cURL, HTTP"}}
+{{partial:tab name="cURL"}}
+```bash
+curl --location --request POST 'https://amplitude.com/api/3/annotation-categories' \
+--header 'Authorization: Basic {api-key}:{secret-key}' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "category": "Releases"
+}'
+```
+{{/partial:tab}}
+{{partial:tab name="HTTP"}}
+```bash
+POST /api/3/annotation-categories HTTP/1.1
+Host: amplitude.com
+Authorization: Basic {api-key}:{secret-key} #credentials must be base64 encoded
+Content-Type: application/json
+
+{
+  "category": "Releases"
+}
+```
+{{/partial:tab}}
+{{/partial:tabs}}
+
+#### Body parameters
+
+| Parameter  | Description                                                               |
+| ---------- | ------------------------------------------------------------------------- |
+| `category` | <span class="required">Required</span>. String. The name of the category. |
+
+#### Response
+
+```json
+{
+  "success": true
```

Comment:
Sorry my bad the response is actually the created category (I fixed it in the API doc):

```
{
   "data": {
      "id": 569,
      "category": "test category api"
     }
}
```